plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/dialogs/driver/EditMavenArtifactDialog.java

```diff
@@ -128,18 +222,152 @@ private void updateButtons() {
         );
     }
 
+    private List<DriverLibraryMavenArtifact> parseGradle() throws XMLException {
+        final List<DriverLibraryMavenArtifact> artifacts = new ArrayList<>();
+        final Matcher matcher = REGEX_FOR_GRADLE.matcher(fieldText.getText());
+        while (matcher.find()) {
+            final String classifier = matcher.group("group");
+
+            // Try to match long form
+            String group = matcher.group("group");
+            String name = matcher.group("name");
+            String version = matcher.group("version");
+
+            if (!CommonUtils.isNotEmpty(group) && !CommonUtils.isNotEmpty(name) && !CommonUtils.isNotEmpty(version)) {
```

Comment:
1. Please get rid of these double negations.
2. It's totally OK for the version to be empty. In this case, we should use the latest version (RELEASE version in our defenitions)